apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -13,6 +13,11 @@ import PopUpMenu from './PopUpMenu';
 import ConfirmEnableMakerDialog from './ConfirmEnableMakerDialog';
 import LibraryManagerDialog from '@cdo/apps/code-studio/components/libraries/LibraryManagerDialog';
 import {getStore} from '../../redux';
+import experiments from '@cdo/apps/util/experiments';
+import loadable from '@cdo/apps/util/loadable';
+const ModelManagerDialog = loadable(() =>
+  import('@cdo/apps/code-studio/components/ModelManagerDialog')
+);
```

Comment:
@davidsbailey Like this? My Storybook issues are solved with this change! Is there anything else?

Reply:
https://user-images.githubusercontent.com/12300669/106693140-d0aceb00-65a3-11eb-9c56-5b27c3600a31.mov

Hmmm... actually, we probably don't want to lazy load this? Or maybe there's a way to solve the dependency issue without the loading spinner?

Reply:
yes, that code looks correct. you might be able to configure the `loadable` library to not show the spinner. if that doesn't fix your problems, then there should be another way to break the dependency -- one way would be to pass a callback to the model manager which calls into applab, rather than hard-coding it to call into applab. or apply a similar strategy at another point in the chain of dependencies.